app/views/payment-links/information.njk

```diff
@@ -94,24 +93,67 @@
           "rows": "5",
           "lang": lang
         }
-      })
-    }}
+      }) }}
+
+      {% set formattingHelpHTML %}
+        <h3 class="govuk-heading-m">Links and URLs</h3>
+
+        <p>To add a link, use square brackets [ ] around the link text, and round brackets ( ) around the full URL. Make
+          sure there are no spaces between the two sets of brackets. For example:</p>
```

Comment:
[Notify have got some really solid Markdown advice in their docs](https://www.notifications.service.gov.uk/using-notify/links-and-URLs). I'd recommend we lift as much as possible from their guidance to help our service users create good content (for example recommending avoiding third party url-shorteners).